core/domain/exp_jobs_one_off_test.py

```diff
@@ -1075,7 +1076,9 @@ def test_for_textangular_validation_and_migration(self):
                 job_id))
 
         # Test that validation passes after migration.
-        self.assertEqual(actual_output, [])
+        # There should be no validation errors in new updated exploration.
+        # There are 16 validation errors of 1st exploration.
+        self.assertEqual(len(actual_output), 16)
```

Comment:
Does this mean that the job runs on both the explorations - original and updated and since we don't overwrite the original one, we get those errors as well?

Reply:
Ok, Thanks.